app/views/individual/claimenrolment/ClaimEnrolmentAlreadySignedUp.scala.html

```diff
@@ -0,0 +1,45 @@
+@*
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import config.AppConfig
+@import views.html.templates.PrincipalMainTemplate
+
+@this(
+
+principalMainTemplate: PrincipalMainTemplate
+
+)
+@()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@principalMainTemplate(title = messages("claim-enrolment.claimAlreadySignedUp.title")) {
+
+<h1 class="govuk-heading-xl">@messages("claim-enrolment.claimAlreadySignedUp.title")</h1>
+<p class="govuk-body" id="claimAlreadySignedInBody">
+    @messages("claim-enrolment.claimAlreadySignedUp.content")
+</p>
+<ul class="govuk-list govuk-list--bullet">
+    <li>
+        <a class="govuk-link" id="checkSaAccount" target="_blank">
+            @messages("claim-enrolment.claimAlreadySignedUp.content.link1")
+        </a>
+    </li>
+    <li>
+        <a class="govuk-link" id="retrieveSaAccountDetails" target="_blank">
+            @messages("claim-enrolment.claimAlreadySignedUp.content.link2")
+        </a>
```

Comment:
```suggestion
        <a class="govuk-link" id="checkSaAccount">
            @messages("claim-enrolment.claimAlreadySignedUp.content.link1")
        </a>
    </li>
    <li>
        <a class="govuk-link" id="retrieveSaAccountDetails">
            @messages("claim-enrolment.claimAlreadySignedUp.content.link2")
        </a>
```
as per prototype, links are not supposed to open in new tab